<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>564</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>7510</width>
    <height>5700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(252, 249, 249);
border-color: rgb(32, 74, 135);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="vertexLabel">
       <property name="styleSheet">
        <string notr="true">color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>Vertex :</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="vertexHorizontalLayout">
       <item>
        <widget class="QLineEdit" name="vertexLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="vertexPushButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(32, 74, 135);
border-color: rgb(32, 74, 135);
</string>
         </property>
         <property name="text">
          <string>Brows</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="vertexTextEdit">
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(0, 0, 0);
border-color: rgb(32, 74, 135);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="edgesLabel">
       <property name="styleSheet">
        <string notr="true">color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>Edges :</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="edgesHorizontalLayout">
       <item>
        <widget class="QLineEdit" name="edgesLineEdit">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(32, 74, 135);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="edgesPushButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(32, 74, 135);
border-color: rgb(32, 74, 135);</string>
         </property>
         <property name="text">
          <string>Brows</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="edgesTextEdit">
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(0, 0, 0);
border-color: rgb(32, 74, 135);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="outputLabel">
       <property name="styleSheet">
        <string notr="true">color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>Output :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="OutputTextEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(23, 5, 97);
background-color: rgb(238, 238, 236);</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="dfsLineEdit">
         <property name="styleSheet">
          <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(0, 0, 0);
border-color: rgb(32, 74, 135);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sortPushButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(32, 74, 135);</string>
         </property>
         <property name="text">
          <string>DFS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGenerate"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGenerate">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/155-Random-512.png</normaloff>
     <normalon>:/155-Random-512.png</normalon>:/155-Random-512.png</iconset>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
   <property name="toolTip">
    <string>Generates random vertices and edges files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
